service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/UploadDraftOrdersAboutToSubmitControllerTest.java

```diff
@@ -41,8 +47,13 @@
 @OverrideAutoConfiguration(enabled = true)
 class UploadDraftOrdersAboutToSubmitControllerTest extends AbstractUploadDraftOrdersControllerTest {
 
+    @MockBean
```

Comment:
ideally we should not mock this one (this is our service)
instead you can mock idamClient to return user with role. Or user RequestData.userRole in DocumentUploadHelper and then see postAboutToStartEvent(CaseData caseData, String... userRoles)